sdk/monitor/Azure.Monitor.Query/src/LogsQueryClient.cs

```diff
@@ -187,6 +191,110 @@ public virtual async Task<Response<LogsBatchQueryResult>> QueryBatchAsync(LogsBa
                 throw;
             }
         }
+
+        /// <summary>
+        /// Create a Kusto query from an interpolated string.  The interpolated values will be quoted and escaped as necessary.
+        /// </summary>
+        /// <param name="filter">An interpolated query string.</param>
+        /// <returns>A valid Kusto query.</returns>
+        public static string CreateQuery(FormattableString filter)
+        {
+            if (filter == null) { return null; }
+
+            string[] args = new string[filter.ArgumentCount];
+            for (int i = 0; i < filter.ArgumentCount; i++)
+            {
+                args[i] = filter.GetArgument(i) switch
+                {
+                    // Null
+                    null => throw new ArgumentException(
+                        $"Unable to convert argument {i} to an Kusto literal. " +
+                        $"Unable to format an untyped null value, please use typed-null expression " +
+                        $"(bool(null), datetime(null), dynamic(null), guid(null), int(null), long(null), real(null), double(null), time(null))"),
+
+                    // Boolean
+                    true => "true",
+                    false => "false",
```

Comment:
Should this be `bool(true|false)`?

Reply:
True and false carry the type information on their own. having the wrapping type is more important where there is parsing or ambiguous type involved.